examples/r-tests.R

```diff
@@ -0,0 +1,8 @@
+setwd("~/github-orgs/zonebuilders/zonebuilder-rust/")
```

Comment:
It seems super weird to assume this is where someone's cloned the repo. Can the R script detect its own location and know that the crate root is one dir up?

Reply:
:+1:

Reply:
Detail on this one: I added it once for the 'reprex' package to run and show the output. Good news, this is now redundant also as the developers of the package have added a `wd` argument: https://reprex.tidyverse.org/reference/reprex.html

It was left in by mistake in any case, well spotted, definitely we're better off without this!